common/scala/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerFactory.scala

```diff
@@ -27,6 +27,8 @@ import scala.concurrent.Future
 
 case class ContainerArgsConfig(network: String,
                                dnsServers: Seq[String] = Seq.empty,
+                               dnsSearch: Seq[String] = Seq.empty,
```

Comment:
Going forward we should look into opening up the docker argument configuration in a generic way such that any argument can be configured without requiring code change

Reply:
I think I could have also done this via extra-args, but since dnsServers was already explicit it seemed better to make the other two dns arguments explicit as well.